extensions/amp-google-assistant-assistjs/0.1/libs/closure-wrapper.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * @fileoverview Export a custom function to create RespondingChannel. The function is basically a wrapper around the real
+ * implementation in Closure library.
+ */
+
+goog.require('goog.messaging.PortChannel');
+goog.require('goog.messaging.RespondingChannel');
+
+function createRC(frameWindow, origin, serviceHandlersMap) {
+  const portChannel =
+      goog.messaging.PortChannel.forEmbeddedWindow(frameWindow, origin);
+  const respondingChannel = new goog.messaging.RespondingChannel(portChannel);
+
+  for (let serviceName of serviceHandlersMap.keys()) {
```

Comment:
Use `forEach`, we specifically disallow for-of iteration because of the necessary `Symbol.iterator` polyfill it adds to the binary.

Reply:
Done.